lib/pengine/tests/utils/Makefile.am

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright 2022 the Pacemaker project contributors
+#
+# The version control history for this file may have further details.
+#
+# This source code is licensed under the GNU General Public License version 2
+# or later (GPLv2+) WITHOUT ANY WARRANTY.
+#
+
+include $(top_srcdir)/mk/tap.mk
+include $(top_srcdir)/mk/unittest.mk
+
+AM_CPPFLAGS += -I$(top_srcdir)/lib/pengine
```

Comment:
Should we include this at a higher level than `tests/utils` to avoid duplication later, or leave it for now since only `utils` needs it currently?

Reply:
probably worthwhile

Reply:
Gonna let Chris decide how he wants this done. Rearranging it, I'm hitting some `include` issues I'd rather not deal with right now